components/AppBar/src/MDCAppBarNavigationController.m

```diff
@@ -64,8 +64,9 @@ - (instancetype)initWithRootViewController:(UIViewController *)rootViewControlle
 }
 
 - (void)MDCAppBarNavigationController_commonInit {
-  // We always want the navigation bar to be hidden.
-  [self setNavigationBarHidden:YES animated:NO];
+  // We always want the UIKit navigation bar to be hidden; to do so we must invoke the super
+  // implementation.
+  [super setNavigationBarHidden:YES animated:NO];
```

Comment:
nit: you can call the API without the animated param if you don't wish to animate.

Reply:
I use the `animate:` variant here mainly to be explicit.

Reply:
ack